src/components/dataTable/icon.scss

```diff
@@ -0,0 +1,37 @@
+table.md-table thead th {
+  md-icon {
+    width: 16px;
+    height: 16px;
+    min-width: 16px;
+    min-height: 16px;
+    font-size: 16px !important;
+    line-height: 16px !important;
+
+    &:not(:first-child) {
+      margin-left: 8px;
+    }
+
+    &:not(:last-child) {
+      margin-right: 8px;
+    }
+
+    &.md-sort-icon {
+      margin-bottom: 1px;
```

Comment:
This makes the icon inline with text. Fonts have extra room at the bottom for characters with tails (e.g 'qgyjp')